PythonAPI/pycocotools/coco.py

```diff
@@ -46,6 +46,8 @@
 
 import json
 import time
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
Can you not enforce the backend of matplotlib?
This would unintentionally set matplotlib backend every time `pycocotools` is imported.